wgsl/index.bs

```diff
@@ -5967,6 +5988,13 @@ See [[#sync-builtin-functions]].
         and discarding the least significant bits.
 
         The number of bits to shift is the value of |e2|, modulo the bit width of |e1|.<br>
+        If |e2| is greater or equal to the bit width or |e1|, then:
+        * It is a [=shader-creation error=] if |e2| is a [=const-expression=].
+        * It is a [=pipeline-creation error=] if |e2| is an [=override-expression=].
+
+        If |e2|+1 most significant bits of |e1| do not have the same bit value, then:
```

Comment:
From #3535  the |e2|+1 condition should hold when e1 is signed.  It should be |e2| when e1 is unsigned.